lib/features/contact/presentation/widgets/contact_item_widget.dart

```diff
@@ -0,0 +1,132 @@
+  const ContactItemWidget({
+    Key? key,
+    required this.emailAddress,
+    required this.selectedContactList,
+    required this.imagePaths,
+    required this.responsiveUtils,
+    required this.onSelectContactAction,
+    this.onDeleteContactAction,
+  }) : super(key: key);
+
+  @override
+  State<ContactItemWidget> createState() => _ContactItemWidgetState();
+}
+
+class _ContactItemWidgetState extends State<ContactItemWidget> {
+
+  bool _isHovering = false;
+
+  @override
+  Widget build(BuildContext context) {
+    final childWidget = Padding(
+      padding: ContactItemWidgetStyle.getItemPadding(
+        context,
+        widget.responsiveUtils
+      ),
+      child: Row(
+        children: [
+          GradientCircleAvatarIcon(
+            colors: widget.emailAddress.avatarColors,
+            label: widget.emailAddress.labelAvatar,
+          ),
+          const SizedBox(width: 12),
+          Expanded(
+            child: Column(
+              mainAxisSize: MainAxisSize.min,
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                Text(
+                  widget.emailAddress.asString(),
+                  maxLines: 1,
+                  overflow: TextOverflow.ellipsis,
+                  style: ContactItemWidgetStyle.nameAddressTextStyle
+                ),
+                if (widget.emailAddress.displayName.isNotEmpty)
+                  Padding(
+                    padding: const EdgeInsets.only(top: 2),
+                    child: Text(
+                      widget.emailAddress.emailAddress,
+                      maxLines: 1,
+                      overflow: TextOverflow.ellipsis,
+                      style: ContactItemWidgetStyle.emailAddressTextStyle
+                    )
+                  )
+              ]
+            )
+          ),
+          if (_isSelectedEmailAddress && widget.onDeleteContactAction == null)
+            Padding(
+              padding: const EdgeInsetsDirectional.symmetric(horizontal: 12),
+              child: SvgPicture.asset(
+                widget.imagePaths.icSelectedSB,
+                width: 20,
+                height: 20,
+                colorFilter: AppColor.primaryColor.asFilter(),
+              ),
+            ),
+          if (PlatformInfo.isWeb && _isHovering && widget.onDeleteContactAction != null)
+            TMailButtonWidget.fromIcon(
+              icon: widget.imagePaths.icClearTextSearch,
+              backgroundColor: Colors.transparent,
+              onTapActionCallback: () =>
+                widget.onDeleteContactAction!(widget.emailAddress),
+            )
+          else if (PlatformInfo.isMobile && widget.onDeleteContactAction != null)
+            TMailButtonWidget.fromIcon(
+              icon: widget.imagePaths.icClearTextSearch,
+              backgroundColor: Colors.transparent,
+              onTapActionCallback: () =>
+                widget.onDeleteContactAction!(widget.emailAddress),
+            )
```

Comment:
Should combine these 2 `if` statements to
```dart
bool get canDeleteContact {
  // ...
}
```

Reply:
Fixed